src/main.js

```diff
@@ -201,6 +201,31 @@ function clearAppCache() {
   }
 }
 
+function restoreWindowState(state) {
```

Comment:
validateWindowPosition sounds as if it would return a bool value, I will go for getValidWindowPosition